secure-biometric-device/README.md

```diff
@@ -0,0 +1,145 @@
+# Secure Biometric Device
+
+A standalone vanillajs library component for interacting with SBI devices and auth capture Face, Finger & Iris detail.
+
+## Build
+
+To build up the library
+
+```bash
+npm run build
+```
+
+To generate a package from it
+
+```bash
+npm pack
+```
+
+For publishing the package
+
+```bash
+npm publish
+```
+
+## Local
+
+### In Html
+
+Either you build the library or serve the libray
+
+```bash
+npm run build
+
+or 
+
+npm run start
+```
+
+After that you can serve the index.html file from the below folder
+```
+secure-biometric-device
+└─── example
+     └─── html
+          │─── index.html
+          └─── mbd.js
+```
+
+How are we implement the library in vanilla javascript, checkout the `mbd.js` file (see above path), below you can see the snippet also.
+
+```js
+SecureBiometricDevice.init({
```

Comment:
Rename SecureBiometricDevice to SecureBiometricInterface

Reply:
hi @vishwa-vyom , this has been done, it can be checked in [12110b1](https://github.com/mosip/esignet-plugins/pull/5/commits/12110b1967a5e97c58d6c20397376a3cae1680c0) commit